morph/react/property-event-handler.js

```diff
@@ -15,7 +15,9 @@ export function enter(node, parent, state) {
   // assuming it is a hook
   if (/^use[A-Z]/.test(node.value)) {
     let variableName = getVariableName(node.name, state)
-    state.variables.push(`let ${variableName} = ${importName}.${node.value}()`)
+    state.variables.push(
+      `let ${variableName} = ${importName}.${node.value}({ viewPath })`
```

Comment:
cool yeah, could we pass the whole set of props? not sure if we have an easy ref to it tbh...

Reply:
I will take a look, not sure how I could get all the props as those are destructured atm

Reply:
maybe we can have a value to replace in here, something like `VIEW_PROPS` and then when the get-body gets the props, it can replace that value with whatever it destructured on the function?

Reply:
ok, will try that. I think it should work